Lib/bdb.py

```diff
@@ -574,6 +574,8 @@ def format_stack_entry(self, frame_lineno, lprefix=': '):
             line = linecache.getline(filename, lineno, frame.f_globals)
             if line:
                 s += lprefix + line.strip()
+        else:
+            s += '\nWarning: lineno is None'
```

Comment:
Is `s += f'{lprefix}Warning: lineno is None'` or `s += lprefix + 'Warning: lineno is None'` better? In `pdb`, `'\n-> '` is used for the prefix. We might want to keep the same format as the correct line printed?

Reply:
```suggestion
            s += f'{lprefix}Warning: lineno is None'
```